src/OSPSuite.R/Services/DataImporterTask.cs

```diff
@@ -17,9 +17,9 @@ public interface IDataImporterTask
    {
       ImporterConfiguration CreateConfiguration();
       ImporterConfiguration GetConfiguration(string filePath);
-      ImporterConfiguration CreateConfigurationFor(string dataPath);
+      ImporterConfiguration CreateConfigurationFor(string dataPath, string sheetName = null);
       void SaveConfiguration(ImporterConfiguration configuration, string path);
-      IReadOnlyList<DataRepository> ImportExcelFromConfiguration(string configurationPath, string dataPath);
+      IReadOnlyList<DataRepository> ImportExcelFromConfiguration(string configurationPath, string dataPath = null);
```

Comment:
Why this change here?

Reply:
So you can define the sheet to use as base for the configuration (similar to reset mappings based on current sheet on the UI) or leave it null and then use the first sheet (default behavior)

Reply:
Why is this called dataPath and not sheet?